main.py

```diff
@@ -256,7 +256,7 @@ async def run(self):
             if not self.game_paused() or is_first_frame:
                 if self.level.cutscene_animation.active:
                     event = pygame.key.get_pressed()
-                    if event[pygame.K_RSHIFT]:
+                    if event[pygame.K_RSHIFT] and self.current_state == GameState.PLAY:
```

Comment:
This check is a bit redundant, because the update call will not have the same effect due to checking if the game state is `PLAY` in the function call

Reply:
yes, you're right - I'll remove it